src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -73,43 +66,46 @@ private Manipulator()
         _twistPID.setInputRange(Constants.Manipulator.TWIST_MIN_ROTATION, Constants.Manipulator.TWIST_MAX_ROTATION);
         _twistPID.setOutputRange(-1, 1);
         _twistPID.setSetpointDeadband(1);
-
-        _graspPID.setCoefficient(Coefficient.P, 0, 0, 0);
-        _graspPID.setCoefficient(Coefficient.I, 0, 0, 0);
-        _graspPID.setCoefficient(Coefficient.D, 0, 0, 0);
-        _graspPID.setInputRange(Constants.Manipulator.GRASP_MIN_POSITION, Constants.Manipulator.GRASP_MAX_POSITION);
-        _graspPID.setOutputRange(-1, 1);
-        _graspPID.setSetpointDeadband(1);
     }
 
     public void setWristAngle(double position)
     {
         _wristPID.setSetpoint(position, getWristAngle());
     }
 
+    public void setWristSpeed(double speed)
+    {
+        _wristMotor.set(speed);
+    }
+
+    public void setWristResetAngle(double position)
```

Comment:
Since we're changing things already, I don't like this function name. Let's call this `zeroWrist`